settings.go

```diff
@@ -210,19 +238,78 @@ func init() {
 	flag.Var(&Settings.modifierConfig.paramHashFilters, "http-param-limiter", "Takes a fraction of requests, consistently taking or rejecting a request based on the FNV32-1A hash of a specific GET param:\n\t gor --input-raw :8080 --output-http staging.com --http-param-limiter user_id:25%")
 }
 
-var previousDebugTime int64
+var previousDebugTime time.Time
 var debugMutex sync.Mutex
+var pID = os.Getpid()
 
-// Debug gets called only if --verbose flag specified
+// Debug take an effect only if --verbose flag specified
 func Debug(args ...interface{}) {
 	if Settings.verbose {
 		debugMutex.Lock()
+		defer debugMutex.Unlock()
 		now := time.Now()
-		diff := float64(now.UnixNano()-previousDebugTime) / 1000000
-		previousDebugTime = now.UnixNano()
-		debugMutex.Unlock()
-
-		fmt.Printf("[DEBUG][PID %d][%d][%fms] ", os.Getpid(), now.UnixNano(), diff)
+		diff := now.Sub(previousDebugTime).String()
```

Comment:
If you leave it as "ns" then in the debug call on line 253 at least say that it is `ns`

Reply:
the built in `Duration.String()` format well by giving it relevant prefix